dev/io.openliberty.webcontainer.servlet.6.1.internal.factories/bnd.bnd

```diff
@@ -102,4 +102,7 @@ Import-Package: \
     com.ibm.ws.resource;version=latest, \
     com.ibm.ws.kernel.boot.core;version=latest,\
     io.openliberty.jakarta.servlet.6.0;version=latest,\
-    io.openliberty.webcontainer.servlet.6.0.internal;version=latest
+    io.openliberty.webcontainer.servlet.6.0.internal;version=latest,\
```

Comment:
Shouldn't this be removed and just using the `io.openliberty.webcontainer.servlet.6.1.internal` project?

Reply:
I think `io.openliberty.webcontainer.servlet.6.0.internal` should stay along with the new `io.openliberty.webcontainer.servlet.6.1.internal`